features/step_definitions/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  rules: {
+    'func-names': 'off',
```

Comment:
This is necessary to prevent ESLint warnings on step definitions, where it makes no sense to give names to every function and we need the `function` expression as we need to access `this`.